### DISCUSSION TITLE
Can't install DistroAV Plugin

### DISCUSSION THREAD

**Discussion by djbassick** [DistroAV/DistroAV#1174] [Q&A - Support]
OBS 31.0.0, Distrav 6.0.0, Brand new installation, Mac Mini M4

DistraAV plugin can't be installed. I get this error.

The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.

Any advice would be appreciated.

Thank you!

**Comment by djbassick** [comment]
![Screenshot 2024-12-28 at 7 33 20 PM](https://github.com/user-attachments/assets/f7f5b9dd-27a2-457a-841b-c15bebf89e59)

**Reply by Trouffman** [reply]
You can check all the steps at : https://www.macobserver.com/tips/how-to/installer-encountered-error-mac/

**Comment by dbetz** [comment]
I had this same problem and tried following the suggestions in the suggested post but was still unable to get the installer to run.

**Comment by Trouffman** [comment]
For anyone with this issue please check the following steps and report if any solved it :

1. Move the files away from the Download folder. New MacOS version are becoming picky on the installer files.
- > Copy the file to the desktop and try again from the desktop to install.

2. Set Privacy & Security Settings > Security > Allow applications from "App Store & Known Developers"
![image](https://github.com/user-attachments/assets/2f966c5d-14ba-43d8-b141-41f576470a85)

**Comment by dbetz** [comment]
I've done both of these things and the install still fails. The odd thing is that the same .pkg file installs fine on my MacBook Pro running the same version of Mac OS X and with the same version of OBS.

**Comment by Trouffman** [comment]
All will be manage in the issue at : https://github.com/DistroAV/DistroAV/issues/1188
